internal/querynode/shard_cluster.go

```diff
@@ -449,10 +449,14 @@ func (sc *ShardCluster) watchSegments(evtCh <-chan segmentEvent) {
 	}
 }
 
-// getNode returns shallow copy of shardNode
-func (sc *ShardCluster) getNode(nodeID int64) (*shardNode, bool) {
+// GetNode returns shallow copy of shardNode
+func (sc *ShardCluster) GetNode(nodeID int64) (*shardNode, bool) {
 	sc.mut.RLock()
 	defer sc.mut.RUnlock()
+	return sc.getNode(nodeID)
+}
+
+func (sc *ShardCluster) getNode(nodeID int64) (*shardNode, bool) {
```

Comment:
Why change get node to public definition? Maybe `getNode` and `getNodeInternal` (forgive my naming sense). The point is this function shall be private